test/foodRoutes.spec.js

```diff
@@ -0,0 +1,85 @@
+const environment = process.env.NODE_ENV || 'test'
+const configuration = require('../knexfile.js')[environment]
+const database = require('knex')(configuration)
+const chai = require('chai')
+const should = chai.should()
+const chaiHttp = require('chai-http')
+const server = require('../app.js')
+
+chai.use(chaiHttp)
+
+describe('API routes', () => {
+
+  before((done) => {
+    database.migrate.latest()
+      .then(() => done())
```

Comment:
After further exploration I'd like to learn how to implement database cleaners, and also figure out why our test environment isn't being utilized, even though its being required properly.